entrypoint.sh

```diff
@@ -613,6 +617,12 @@ prepare() { local mode no_tap=1
 		no_tap=0 # we want subtests
 	fi
 
+	cat <<EOF > "${BASH_PROFILE}"
```

Comment:
I guess it is fine to override this file all the time, it is not common for a user to modify this file I think. So all good!